custom_components/mqtt_vacuum_camera/__init__.py

```
@@ -18,7 +18,7 @@
 from homeassistant.helpers.reload import async_register_admin_service
 from homeassistant.helpers.storage import STORAGE_DIR
 
-from .common import get_vacuum_device_info, get_vacuum_mqtt_topic
+from .common import get_vacuum_device_info, get_vacuum_mqtt_topic, update_options
```

Ruff (0.8.2)

`.common.update_options` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)